modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/client/DefaultExtensionsClient.java

```diff
@@ -147,7 +146,7 @@ private OperationMessageProcessor createProcessor(String extensionName, String o
         }
       } else {
         if (value instanceof String && parser.isContainsTemplate((String) value)) {
-          values.put(name, new TypeSafeExpressionValueResolver<>((String) value, Object.class));
+          values.put(name, new ExpressionValueResolver((String) value));
```

Comment:
because you are wanting to transform nothing, request an `Object` is like request anything because no transformation will occur